corehq/motech/repeaters/static/repeaters/js/bootstrap3/repeat_record_report.js

```diff
@@ -1,349 +1,342 @@
+import "commcarehq";
```

Comment:
I'm curious whether it's convention, or another reason, for putting "commcarehq" first?

Reply:
It's documented as required (see [here](https://commcare-hq.readthedocs.io/js-guide/dependencies.html#how-do-i-create-a-new-page-with-javascript)) although I don't know why. @biyeun will know.